src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Sql/Sprocs/RebuildIndex_2.sql

```diff
@@ -0,0 +1,60 @@
+﻿/*************************************************************
+    Stored procedures for rebuild index
+**************************************************************/
+--
+-- STORED PROCEDURE
+--     RebuildIndex
+--
+-- DESCRIPTION
+--     Stored procedures for rebuild index
+--
+-- PARAMETERS
+--     @tableName
+--         * index table name
+--     @indexName
+--         * index name
+--     @pageCompression
+--         * index page compression
+
+GO
+CREATE PROCEDURE dbo.RebuildIndex_2
+    @tableName nvarchar(128),
+    @indexName nvarchar(128),
+    @pageCompression bit
```

Comment:
>@pageCompression bit [](http://example.com/codeflow?start=4&length=20)

Please add default = 0 and use previous RebuildIndex stored procedure (no need to add new one).

Reply:
Thanks, updated.